source/deployment/bulk-loading-data.rst

```diff
@@ -28,3 +28,6 @@ Before running the bulk loading command, you must first create a `JSONL
   ``sudo ./mattermost import bulk data.jsonl --apply``
 
 5. When the bulk load command completes, clear all caches. Open the System Console, and click **General > Configuration > Purge All Caches**.
+
+.. important::
+  Owner for ``data`` dir has to change to Mattermost user for it to work correctly. Otherwise, Mattermost cannot fetch the files created in ``data`` dir after the import tool has run, since all files created in ``data`` dir are owned by ``root`` as the tool was run as ``sudo``.
```

Comment:
Actually, this must be something like "Owner for ``data`` directory and all its content has to change to Mattermost user for it to work correctly.". The rest of the text looks good to me.

Reply:
Thanks, updated.